packages/js/components/src/advanced-filters/index.js

```diff
@@ -320,7 +320,9 @@ class AdvancedFilters extends Component {
 						<div className="woocommerce-filters-advanced__add-filter">
 							<Dropdown
 								className="woocommerce-filters-advanced__add-filter-dropdown"
-								position="bottom center"
+								popoverProps={ {
+									position: 'bottom center',
+								} }
```

Comment:
> Note: use the placement prop instead when possible.
> [position prop](https://developer.wordpress.org/block-editor/reference-guides/components/popover/#position-yaxis-xaxis-optionalcorner)

Something like the following (not tested):
```suggestion
								popoverProps={ {
									placement: 'bottom',
								} }
```